core_backend/app/__init__.py

```diff
@@ -26,6 +26,35 @@
 
 logger = setup_logger()
 
+page_description = """
+Welcome to the API documentation for the Ask A Question backend. These API is used to
+interact with the Ask A Question application.
+
+The important endpoints here are divided into the following two groups:
+
+1. APIs to be called from your chat manager (authenticated via API key):
+- **Question-answering and feedback collection**: LLM-powered question answering based
+on your content. Plus feedback collection for the answers.
+- **Urgency detection**: Detect urgent messages according to your urgency rules.
+
+2. APIs used by the AAQ Admin App (authenticated via user login):
+- **Question-answering content management**: APIs to manage the contents in the
+application.
+- **Question-answering content tag management**: APIs to manage the content tags in the
+application.
+- **Urgency rules management**: APIs to manage the urgency rules in the application.
+"""
+tags_metadata = [
+    question_answer.TAG_METADATA,
+    urgency_detection.TAG_METADATA,
+    contents.TAG_METADATA,
+    tags.TAG_METADATA,
+    urgency_rules.TAG_METADATA,
+    dashboard.TAG_METADATA,
+    auth.TAG_METADATA,
+    user_tools.TAG_METADATA,
+    admin.TAG_METADATA,
+]
```

Comment:
I reordered these to reflect people's usecase when they come to this page. also corresponds with description above. what do you think?

Reply:
Sure ! Thank you for the page description also, I didn't know about this param!

I had put content first because question_answering is not useful without content. But this ordering also makes sense!